ViewGenerator/build/ViewGenerator.targets

```diff
@@ -49,15 +49,15 @@
 
   <Target Name="VG_CleanLegacyJSFiles" >
     <ItemGroup>
-      <JSFilesToDelete Include="**\*.js" Exclude="node_modules\**\*.js"/>
+      <JSFilesToDelete Include="**\*.js" Exclude="node_modules\**\*.js;$(CleanFilesExcludePath)**\*.js"/>
     </ItemGroup>
     <Message Condition="'@(JSFilesToDelete)' != ''" Text="Deleting old files: @(JSFilesToDelete)" Importance="high" />
     <Delete Files="@(JSFilesToDelete)"/>
   </Target>
 
   <Target Name="VG_CleanLegacyCSSFiles" >
```

Comment:
I know that this was already like this, but can you change these targets name from `VG_CleanLegacy<Type>Files` to `VG_Clean<Type>Files`?

Reply:
:+1: